api/v1/api.md

```diff
@@ -0,0 +1,151 @@
+# dcrtime API Specificiation
+
+V1
+
+This document describes the REST API provided by a `dcrtimed` server.  This API allows users to create upload hashes which are periodically submitted to the decred blockchain and timestamped as well as to confirm such timestamps.
+
+**Methods**
+
+- [`Timestamp`](#timestamp)
+- [`Verify`](#verify)
+
+**Return Codes**
+
+- [`ResultOK`](#ResultOK)
+- [`ResultExistsError`](#ResultExistsError)
+- [`ResultDoesntExistsError`](#ResultDoesntExistsError)
+- [`ResultDisabled`](#ResultDisabled)
+
+### Methods
+
+#### `Timestamp`
+
+Query the server to store one or more digests and return a collection the will contain the supplied digest.
```

Comment:
This sentence doesn't flow.  Not even sure what you are trying to say.  We really could use a few more sentences describing this too.